.gitignore

```diff
@@ -0,0 +1,69 @@
+# Logs
```

Comment:
I am not sure all required files/directories are added here. use gitignore.io to generate the files
keywords: gatsby yarn firebase node